packages/odata/src/services/odataQueryBuilder.service.ts

```diff
@@ -7,6 +7,11 @@ export class OdataQueryBuilderService {
   _filterCount = 0;
   _odataOptions: Partial<OdataOption>;
 
+  private _columnDefinitions: Column[] = [];
```

Comment:
I using more `protected` than `private` nowadays so it can be extended (that was actually a request from you last year lol)